test/integration/sequelize/query.test.js

```diff
@@ -56,7 +63,9 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
       await this.sequelize.query(this.insertQuery);
     });
 
-    it('executes a query if a placeholder value is an array', async function() {
+    // Oracle dialect doesn't support insert of multiple rows using insert into statement
+    // INSERT ALL INTO statement can be used instead
+    (dialect !== 'oracle' ? it : it.skip)('executes a query if a placeholder value is an array', async function() {
```

Comment:
can you tweak the test to make it work with oracle too? It's important to test whether `replacements` is handled properly

Reply:
Added a sql specific to Oracle dialect to make this test work.